release-notes/latest.md

```diff
@@ -1,3 +1,7 @@
+#### <sub><sup><a name="4535" href="#4535">:link:</a></sup></sub> feature, breaking
+
+* The cf auth connector used to only return spaces for the `developer` role. Now it returns spaces for `developer`, `auditor`, and `manager`.
```

Comment:
Might be better to show the user-level impact of this since it's a breaking change:

```suggestion
* The CloudFoundry auth connector, when configured to authorize users based on CF space membership, will now authorize space auditors and space managers in addition to space developers. This is a breaking change as any teams with CF space-based configuration may grant access to users that they wouldn't have before. #4661
```

Is that accurate? /cc @xtremerui

Reply:
I think it is good as a warn to ops to double check the granted access.